### ISSUE TITLE
tigerbrew refuses to use gcc5 over gcc

### ISSUE THREAD

**Issue by jeisom** [mistydemeo/tigerbrew#527]
I get the below when I try to use gcc5 and --cc=gcc-5

```
Error: The requested Homebrew GCC was not installed. You must:
 brew install gcc
 
```
however "brew unlink gcc5 && brew link gcc" works with --cc=gcc-5

**Comment by mistydemeo** [comment]
Can you confirm what version of `gcc5` you have installed? I can't repro with the latest gcc5 (5.4.0).

**Comment by jeisom** [comment]
I am not near my Mac, but would Spotlight being disabled be a/the problem?

Later

Jonathan

From: Misty De Meo <notifications@github.com> <notifications@github.com>
Reply: mistydemeo/tigerbrew
<reply@reply.github.com>
<reply@reply.github.com>
Date: January 24, 2018 at 2:32:46 PM
To: mistydemeo/tigerbrew <tigerbrew@noreply.github.com>
<tigerbrew@noreply.github.com>
CC: jeisom <jeisom@gmail.com> <jeisom@gmail.com>, Author
<author@noreply.github.com> <author@noreply.github.com>
Subject:  Re: [mistydemeo/tigerbrew] tigerbrew refuses to use gcc5 over gcc
(#527)

Can you confirm what version of gcc5 you have installed? I can't repro with

**Comment by mistydemeo** [comment]
Nope, not in this case! Homebrew doesn't need it to determine versions of Homebrew-installed software.
